test/RiskManagerV1.test.js

```diff
@@ -71,28 +72,47 @@ describe("RiskManagerV1", () => {
       })
     })
 
-    context("when deposit is in liquidation state", () => {
-      let notifyLiquidationTx
-
-      beforeEach(async () => {
-        notifyLiquidationTx = await notifyLiquidation()
+    context("when deposit is above collateralization threshold level", () => {
+      it("should revert", async () => {
+        await mockIDeposit.mock.currentState.returns(
+          depositLiquidationInProgressState
+        )
+        await mockIDeposit.mock.collateralizationPercentage.returns(
+          collateralizationThreshold + 1
+        )
+        await expect(
+          riskManagerV1.notifyLiquidation(mockIDeposit.address)
+        ).to.be.revertedWith(
+          "Deposit collateralization is above the threshold level"
+        )
       })
+    })
 
-      it("should emit NotifiedLiquidation event", async () => {
-        await expect(notifyLiquidationTx)
-          .to.emit(riskManagerV1, "NotifiedLiquidation")
-          .withArgs(mockIDeposit.address, notifier.address)
-      })
+    context(
+      "when deposit is in liquidation state and not above collateralization threshold",
```

Comment:
`s/and not above collateralization threshold/and at the collateralization threshold` maybe?

Reply:
404eebe